### ISSUE TITLE
"NotSupportedException: The URI prefix is not recognized" on MacOS build with .NET 4.6 backend

### ISSUE THREAD

**Issue by Bunkerbewohner** [stomt/unity#13]
Hi, so this is a weird one. On MacOS, and only on MacOS, when I "build and run" a Unity project with STOMT I get the following error:

```
NotSupportedException: The URI prefix is not recognized.
  at System.Net.WebRequest.Create (System.Uri requestUri, System.Boolean useUriBase) [0x000db] in <f044fe2c9e7e4b8e91984b41f0cf0b04>:0 
  at System.Net.WebRequest.Create (System.String requestUriString) [0x00017] in <f044fe2c9e7e4b8e91984b41f0cf0b04>:0 
  at Stomt.StomtAPI.WebRequest (System.String method, System.String url) [0x00002] in G:\Quiz\Repo\Assets\stomt Collect Player-Feedback\Scripts\StomtAPI.cs:218 
  at Stomt.StomtAPI+<LoadTarget>c__Iterator0.MoveNext () [0x00049] in G:\Quiz\Repo\Assets\stomt Collect Player-Feedback\Scripts\StomtAPI.cs:234 
  at UnityEngine.SetupCoroutine.InvokeMoveNext (System.Collections.IEnumerator enumerator, System.IntPtr returnValueAddress) [0x00028] in /Users/builduser/buildslave/unity/build/Runtime/Export/Coroutines.cs:17 
UnityEngine.MonoBehaviour:StartCoroutine_Auto_Internal(MonoBehaviour, IEnumerator)
UnityEngine.MonoBehaviour:StartCoroutine(IEnumerator) (at /Users/builduser/buildslave/unity/build/artifacts/generated/common/runtime/MonoBehaviourBindings.gen.cs:62)
Stomt.StomtAPI:Start() (at G:\Quiz\Repo\Assets\stomt Collect Player-Feedback\Scripts\StomtAPI.cs:213)
```

I looked at the code and my first guess was that somehow *restServerUrl* wasn't set properly. However, I added some log statements immediately before the WebRequest call in StomtAPI.cs:234 and the value was "https://rest.stomt.com" as expected. 

Also the error only started showing up after upgrading to the newest version from the Asset store. On Windows and Linux the error does not appear in builds or editor. The error does also not appear on MacOS when the code is run in the editor.

*The error only appears with the new .NET 4.6 backend, not with the old .NET 2.0 one.*

I attached an example project that includes nothing but the STOMT plugin on a Canvas which reproduces the error for me:
[Stomt MacOS Test.zip](https://github.com/stomt/stomt-unity-sdk/files/1361215/Stomt.MacOS.Test.zip)

*System Info*
MacOS Sierra 10.12.6
![screen shot 2017-10-06 at 02 23 03](https://user-images.githubusercontent.com/500529/31258145-64819a9a-aa3d-11e7-9355-2b3d937bdf91.png)

**Comment by maxklenk** [comment]
Hi @Bunkerbewohner, it took us a while to reproduce this strange behavior. As you already mentioned the url is build right.

It seems to be a known Unity bug which will be fixed in the 2017.3.0b1 release.
https://issuetracker.unity3d.com/issues/dot-net-4-dot-6-ios-build-throws-notsupportedexception-when-creating-a-webrequest-via-system-dot-net-dot-webrequest-dot-create

Sadly the new Unity versions always come with a lot of bugs.

**Comment by Bunkerbewohner** [comment]
Thanks for having a look! I guess then I need to disable STOMT on Mac until that's available. Too bad!
I could probably work around it by using UnityWebRequest instead of System.Net.WebRequest in StomtAPI.cs, but I'm too lazy. :D
